app/components/Settings/scheduler/login-modal.jsx

```diff
@@ -16,21 +17,27 @@ const PromptLoginModal = props => (
             </button>
             <Row className="header">
                 <p>
-                    To create a scheduled query, you'll need to be logged into Plotly. Logging in will reset your query,
-                    so please save it elsewhere before doing so.
+                    To create a scheduled query, you'll need to be logged into Plotly.<br/><br/>
+                    Note: logging in will reset your query, click the button below to copy the query to your clipboard.
                 </p>
             </Row>
-            <Row>
-                <button type="submit" className="submit" onClick={props.onSubmit}>
+            <Row className="actions">
+                <button type="submit" onClick={props.onSubmit}>
                     Log In
                 </button>
+                <CopyToClipboard text={props.preview.code}>
```

Comment:
Nice!

When I tested it, I noticed the button doesn't give any visual feedback that the copy to clipboard succeeded.

Reply:
+1. would be nice to get an indication that copying has suceeded